<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterLaden</class>
 <widget class="QMainWindow" name="RasterLaden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resonanz Fit - Messung einlesen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>305</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticallayout">
     <item>
      <layout class="QHBoxLayout" name="horizontallayout2">
       <item>
        <widget class="QLineEdit" name="edit_pfad">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_aendern">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ã„ndern...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalspacer0">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="check_konfig">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Messkonfiguration aus Datei einlesen</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontallayout1">
       <item>
        <layout class="QGridLayout" name="gridlayout0">
         <item row="0" column="1">
          <widget class="QSpinBox" name="box_pixel">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_messpunkte">
           <property name="text">
            <string>Messpunkte pro Pixel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="box_messpunkte">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_pixel">
           <property name="text">
            <string>Pixel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalspacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridlayout1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_fmin">
           <property name="text">
            <string>Freq Min (kHz)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_fmax">
           <property name="text">
            <string>Freq Max (kHz)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="box_fmin">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="box_fmax">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="maximum">
            <double>999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalspacer1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontallayout4">
       <item>
        <widget class="QLabel" name="label_methode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Lorentzverteilung</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="box_methode">
         <item>
          <property name="text">
           <string>Antreibendes System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cantilever</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_savgol">
       <property name="text">
        <string>Savitzky-Golay-Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontallayout3">
       <item>
        <widget class="QLabel" name="label_fenster">
         <property name="text">
          <string>Koeffizientenanzahl</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="box_fenster">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ordnung">
         <property name="text">
          <string>Ordnung des Polynoms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="box_ordnung">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalspacer2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontallayout0">
       <item>
        <widget class="QPushButton" name="button_fitten">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Fitten</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
